extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -393,20 +407,47 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 			}
 		});
 
+		this._saveTemplateLink = view.modelBuilder.hyperlink().withProps({
+			label: constants.GENERATE_ARM_TEMPLATE,
+			url: '',
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'font-weight': '400',
+				'line-height': '18px',
+				'text-decoration': 'none',
+				//'margin-right': '8px',
+				'display': 'none',
+			}
+		}).component();
+
+		this._saveTemplateLink.onDidClick(async () => {
+			if (hasRecommendations(this.migrationStateModel)) {
+				const generateProvisioningScriptDialog = new GenerateProvisioningScriptDialog(this.migrationStateModel, this.migrationTargetType);
+				await generateProvisioningScriptDialog.openDialog();
+			}
+		});
+
+		this._separator = view.modelBuilder.text().withProps({
+			value: "|",
+			height: 18,
+			CSSStyles: {
+				'font-size': '15px',
+				'line-height': '18px',
+				'font-weight': '400',
+				'margin': '0px',
```

Comment:
check if this style exist or can be extracted out in styles.ts

Reply:
done